src/Sylius/Behat/Context/Ui/Shop/RegistrationContext.php

```diff
@@ -227,6 +259,138 @@ public function iShouldNotBeLoggedIn()
     }
 
     /**
+     * @When I register with email :email and password :password
+     */
+    public function iRegisterWithEmailAndPassword($email, $password)
+    {
+        $this->registerPage->open();
+        $this->registerPage->specifyEmail($email);
+        $this->registerPage->specifyPassword($password);
+        $this->registerPage->verifyPassword($password);
+        $this->registerPage->specifyFirstName('Carrot');
+        $this->registerPage->specifyLastName('Ironfoundersson');
+        $this->registerPage->register();
+    }
+
+    /**
+     * @Then my account should be verified
+     */
+    public function myAccountShouldBeVerified()
+    {
+        $user = $this->sharedStorage->get('user');
+        $this->securityService->logIn($user->getEmail());
+
+        Assert::true(
+            $this->dashboardPage->isVerified(),
+            'You should be verified.'
```

Comment:
I don't really see any sense, it's an error and it tells you explicitly what should have happened.
On the other hand 👍  for changing `You` => `My account`